web-common/src/runtime-client/http-request-queue/HttpRequestQueue.ts

```diff
@@ -34,6 +34,7 @@ try {
   }
 } catch (err) {
   // no-op
```

Comment:
No longer a no-op. It also looks like this code could be simplied. Does this mean the queue should always be 200?

Reply:
This was for testing. Will revert.